example/sinatra/webapp.rb

```diff
@@ -0,0 +1,77 @@
+require 'veritrans'
+require 'json'
+require 'sinatra'
+
+Veritrans.setup do
+  config.load_yml File.dirname(__FILE__) + "/veritrans.yml#development"
+end
+
+# Alternative way to to initialize core api client:
+# mt_client = Midtrans.new(
```

Comment:
Inconsistent: at above `Veritrans.setup` is used,  then below ` mt_client = Midtrans.new`. please be consistent.